src/grafana/prometheus/docker/prometheus-local/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM prom/prometheus:v2.10.0
+FROM prom/prometheus:v2.11.1
```

Comment:
There is a 2.12.0 tag available -> https://hub.docker.com/r/prom/prometheus/tags -- maybe use that?